ThunkLibs/include/common/Host.h

```diff
@@ -210,6 +337,14 @@ struct GuestWrapperForHostFunction<Result(Args...)> {
   }
 };
 
+// TODO: Move?
```

Comment:
Double-check this. (Should just be moved to a different position in the same file)